Source/ACE/Entity/Door.cs

```diff
@@ -21,30 +24,120 @@ public Door(AceObject aceO)
             : base(aceO)
         {
             PhysicsState |= PhysicsState.HasPhysicsBsp | PhysicsState.ReportCollision;
-            CurrentMotionState = motionStateClosed;
+            if (!DefaultOpen)
+            {
+                CurrentMotionState = motionStateClosed;
+                IsOpen = false;
+                Ethereal = false;
+            }
+            else
+            {
+                CurrentMotionState = motionStateOpen;
+                IsOpen = true;
+                Ethereal = true;
+            }
+
+            // If we had the base weenies this would be the way to go
+            ////if (DefaultLocked)
+            ////    IsLocked = true;
+            ////else
+            ////    IsLocked = false;
+
+            // But since we don't know what doors were DefaultLocked, let's assume for now that any door that starts Locked should default as such.
+            if (IsLocked)
+                DefaultLocked = true;
+
             movementOpen.ForwardCommand = (ushort)MotionCommand.On;
             movementClosed.ForwardCommand = (ushort)MotionCommand.Off;
         }
 
-        public override void OnUse(ObjectGuid playerId)
+        public bool Ethereal
         {
-            // TODO: check if door is locked, send locked soundfx if locked and fail to open.
-            float autoCloseTime = 30.0f;
+            get { return AceObject.GetBoolProperty(PropertyBool.Ethereal) ?? false; }
+            set { AceObject.SetBoolProperty(PropertyBool.Ethereal, value); }
+        }
 
-            if (this.CurrentMotionState == motionStateClosed)
-            {
-                Open();
+        public bool IsOpen
+        {
+            get { return AceObject.GetBoolProperty(PropertyBool.Open) ?? false; }
+            set { AceObject.SetBoolProperty(PropertyBool.Open, value); }
+        }
 
-                // Create Door auto close timer
-                ActionChain autoCloseTimer = new ActionChain();
-                autoCloseTimer.AddDelaySeconds(autoCloseTime);
-                autoCloseTimer.AddAction(this, () => Close());
-                autoCloseTimer.EnqueueChain();
-            }
-            else
-            {
-                Close();
-            }
+        public bool IsLocked
```

Comment:
I'm betting we don't need to save "IsLocked" in the database.  If something defaults to locked, we spawn it locked until somebody unlocks it.